charts/super-agent/charts/super-agent-deployment/tests/configmap_fleet_configs_test.yaml

```diff
@@ -9,12 +9,12 @@ tests:
     set:
       config:
         subAgents:
-          open-telemetry: null
+          open-telemetry: ~
```

Comment:
what does it mean? can you add a comment?

Reply:
It is a left over. I was testing if `null` was being unmarshalled as a string instead of literally `null`.

`~` is the literal `null` in YAML as the spec says: https://yaml.org/type/null.html

> Shorthand:
> > `!!null`
> 
> Kind:
> > `Scalar`
> 
> Canonical:
>  > `~`
> 
> Regexp:
> > ```
> >      ~ # (canonical)
> >     |null|Null|NULL # (English)
> >     | # (Empty)
> > ```